### DISCUSSION TITLE
Fatal error in internal_Init_thread: Other MPI error, error stack:

### DISCUSSION THREAD

**Discussion by js-jixu** [idaholab/moose#23610] [Q&A General] [answered]
Hi, developers.

I installed moose on a cloud host. But moose can't run in parallel:

```
Fatal error in internal_Init_thread: Other MPI error, error stack:
internal_Init_thread(60)...........: MPI_Init_thread(argc=0x7fff718e6c6c, argv=0x7fff718e6c60, required=0, provided=0x7fff718e6c7c) failed
MPII_Init_thread(209)..............: 
MPID_Init(75)......................: 
init_world(190)....................: channel initialization failed
MPIDI_CH3_Init(84).................: 
MPID_nem_init(313).................: 
MPID_nem_tcp_init(175).............: 
MPID_nem_tcp_get_business_card(397): 
GetSockInterfaceAddr(369)..........: gethostbyname failed, simple (errno 1)
```

The installation process looks very smooth, and there are no errors. What is the reason?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Please see this troubleshooting page for this particular error message

https://mooseframework.inl.gov/help/troubleshooting.html#6790d8ab-b8fe-4f9d-bc56-04386260d3b1

Guillaume

**Reply by js-jixu** [reply]
Hi, Guillaume.

I've read the troubleshooting page but failed to find related information about my problem. I can activate moose and run the input file in serial. But when I use `mpiexec -n 2 ~/projects/.../opt -i `, it encounters an error.

This virtual machine has previously installed openmc. When I use `conda info -- envs` to check the environment, it shows this:

![e09021567f4e602a18e7a8f944cbd3c](https://user-images.githubusercontent.com/99813361/222968203-fce604c9-780e-4d8c-bda7-89e20e5609a2.png)

And I've found that there is nothing in `/root/mambaforge3/envs`, while there is a `moose` folder in `/root/mambaforge3/envs` on my own PC's moose. Is it because I didn't install moose in the anaconda3 environment？

**Reply by GiudGiud** [reply]
Did you search for gethostbyname on that page?
It s about modifying the hosts file

**Reply by js-jixu** [reply]
Your suggestion is very helpful and I can run it in parallel!
